charts/internal/gvisor/values.yaml

```diff
@@ -1,4 +1,4 @@
 config:
-  kubernetesVersion: 1.16.1
+  kubernetesVersion: 1.24.4
```

Comment:
It looks like `config.kubernetesVersion` is no longer used after this change -> we could completely clean it up.

```suggestion
```

https://github.com/gardener/gardener-extension-runtime-gvisor/blob/00c4cced57b8facf4935a66a6dc832c46838b5c4/pkg/charts/values.go#L61-L63 as to be cleaned up as well.